web/src/locales/en.json

```
@@ -144,5 +147,16 @@
 	"employees.updatePassword.error.incorrectPassword.description": "The password entered does not match your current password.",
 	"employees.error.passwordMismatch": "The password doesn't match the new password",
 	"employees.passwordConstraints": "Password must contain at least 14 characters, including one regular character, one special character, and one number",
-	"confirm": "Confirm"
+	"employees.dateOfBirth.placeholder": "Enter birthdate",
+	"employees.scheduleStart.placeholder": "Enter start time of schedule",
+	"employees.scheduleEnd.placeholder": "Enter end time of schedule",
+	"employees.create.title": "Add employee",
+	"employees.create.success": "Employee added successfully",
+	"employees.role.placeholder": "Select a role",
+	"employees.password.placeholder": "Enter password",
+	"employees.security": "Security",
+	"employees.password.confirm.placeholder": "Repeat the password",
+	"employees.password.error.passwordMismatch": "The password doesn't match the password",
```

This sounds redundant:

```suggestion
	"employees.password.error.passwordMismatch": "The password doesn't match",
```